index.html

```diff
@@ -5,9 +5,12 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
 
-    <link href="//fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
+    <link href="http://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
```

Comment:
Yeah, I was running locally so I was getting file:// not found. I have never seen this type of link. 

Changed to https.